src/box.jl

```diff
@@ -18,6 +18,9 @@ Box(c::SVector{N}, d::SVector{N},
 Box(c::AbstractVector, d::AbstractVector, axes=eye(length(c)), data=nothing) =
     (N = length(c); Box(SVector{N}(c), SVector{N}(d), SMatrix{N,N}(axes), data))
 
+Box(b::NTuple{2,AbstractVector}, axes=eye(length(b[1])), data=nothing) =
```

Comment:
It would be clearer to just define a new function, like`boxdiag(c::AbstractVector, d::AbstractVector, axes=eye(length(c)), data=nothing)`, rather than defining a constructor that takes a tuple of points.